code/game/machinery/computer/Operating.dm

```diff
@@ -7,18 +7,19 @@
 	icon_keyboard = "med_key"
 	icon_screen = "crew"
 	circuit = /obj/item/circuitboard/operating
-	var/obj/machinery/optable/table = null
-	var/mob/living/carbon/human/victim = null
+	var/obj/machinery/optable/table
 	light_color = LIGHT_COLOR_PURE_BLUE
-	var/verbose = 1 //general speaker toggle
-	var/patientName = null
+	var/verbose = TRUE //general speaker toggle
 	var/oxyAlarm = 30 //oxy damage at which the computer will beep
-	var/choice = 0 //just for going into and out of the options menu
-	var/healthAnnounce = 1 //healther announcer toggle
-	var/crit = 1 //crit beeping toggle
+	var/choice = FALSE //just for going into and out of the options menu
+	var/healthAnnounce = TRUE //healther announcer toggle
+	var/crit = TRUE //crit beeping toggle
 	var/nextTick = OP_COMPUTER_COOLDOWN
 	var/healthAlarm = 50
-	var/oxy = 1 //oxygen beeping toggle
+	var/oxy = TRUE //oxygen beeping toggle
+	var/newPat = TRUE
```

Comment:
Let's not have a "this thing has changed - please update me" var.
Either call a "please update your state" proc, or let the machinery itself notice that things have changed (like it used to do with `if(patientName!=table.victim.name)`

Reply:
patientName is exactly what that is.